src/async-wrap.cc

```diff
@@ -181,6 +183,33 @@ static void Initialize(Local<Object> target,
 }
 
 
+void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
+  Environment* env = Environment::from_destroy_ids_idle_handle(handle);
```

Comment:
So, I tried this and it surprisingly didn't abort:
```diff
diff --git a/src/async-wrap.cc b/src/async-wrap.cc
index a06e163..3e57ee8 100644
--- a/src/async-wrap.cc
+++ b/src/async-wrap.cc
@@ -185,6 +185,8 @@ void AsyncWrap::Initialize(Local<Object> target,
 
 void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
   Environment* env = Environment::from_destroy_ids_idle_handle(handle);
+  v8::SealHandleScope seal(env->isolate());
+  v8::Context::Scope context_scope(env->context());
   // This doesn't leak a handle.
   Local<Function> fn = env->async_hooks_destroy_function();
   int64_t current_id;
@@ -194,11 +196,12 @@ void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
   if (fn.IsEmpty())
     return env->erase_destroy_ids();
 
+  TryCatch try_catch(env->isolate());
+
   while (env->destroy_ids_length() > 0) {
     current_id = env->pop_destroy_id();
     HandleScope scope(env->isolate());
     Local<Value> argv = Number::New(env->isolate(), current_id);
-    TryCatch try_catch(env->isolate());
     MaybeLocal<Value> ret = fn->Call(
         env->context(), Undefined(env->isolate()), 1, &argv);
 
```
Which surprises me a bit, but leads me to believe it isn't leaking a handle not wrapping the `TryCatch` in a `HandleScope`. Guess since it's using RAII it doesn't need the `HandleScope` to clean up?